src/app/modules/shared/components/month-picker/month-picker.component.html

```diff
@@ -1,14 +1,21 @@
 <div class="card-header">
-  <div class="d-flex align-items-center">
-    <button class="btn mr-auto" (click)="changeYear('subtract')">
-      <i class="fa fa-chevron-left"></i>
-    </button>
-    <p>
-      {{selectedYearText}}
-    </p>
-    <button class="btn ml-auto" (click)="changeYear('add')">
-      <i class="fa fa-chevron-right"></i>
-    </button>
+  <div class="row align-items-center">
+    <div class="col-sm-4">
+      <button class="btn mr-auto" (click)="changeYear('subtract')">
+        <i class="fa fa-chevron-left"></i>
+      </button>
+    </div>
+    <div class="col-sm-4">
+      <p class="text-center year-text">
+        {{selectedYearText}}
+      </p>
+    </div>
+
+    <div class="col-sm-4 text-right">
+      <button *ngIf="selectedYear < currentYear" class="btn ml-auto" (click)="changeYear('add')">
```

Comment:
Maybe we would need a test for this. I don't know how hard would be to test in this file, or maybe we could put the logic in the TS file?

Reply:
I go to check it.